gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/gc/GcProfile.java

```diff
@@ -46,7 +47,18 @@
         private int initiatingOccupancyFraction = 68;
 
         @Override
-        public final List<String> gcJvmOpts() {
+        public final List<String> gcJvmOpts(JavaVersion javaVersion) {
+            // TODO(forozco): Update to use proper constant once we can support gradle 6.0
+            if (javaVersion == JavaVersion.VERSION_HIGHER) {
+                return ImmutableList.of(
+                        "-XX:+UnlockExperimentalVMOptions",
+                        // https://wiki.openjdk.java.net/display/shenandoah/Main
+                        "-XX:+UseShenandoahGC",
+                        // "forces concurrent cycle instead of Full GC on System.gc()"
+                        "-XX:+ExplicitGCInvokesConcurrent",
+                        "-XX:+ClassUnloadingWithConcurrentMark",
+                        "-XX+UseNUMA");
+            }
```

Comment:
Might be worth calling out in the changelog that opting into `gc 'response-time'` now gets you shenandoah if you're intending to run on Java13+?